src/main/java/com/internet/shop/security/AuthenticationServiceServiceImpl.java

```diff
@@ -0,0 +1,25 @@
+package com.internet.shop.security;
+
+import com.internet.shop.exceptions.AuthenticationException;
+import com.internet.shop.lib.Inject;
+import com.internet.shop.lib.Service;
+import com.internet.shop.model.User;
+import com.internet.shop.service.UserService;
+
+@Service
+public class AuthenticationServiceServiceImpl implements AuthenticationService {
+    @Inject
+    private UserService userService;
+
+    @Override
+    public User login(String login, String password)
+            throws AuthenticationException {
+        User userFromDB = userService.findByLogin(login).get();
```

Comment:
is there is no value in Optional, you will get the Exception here

let's have this variable of type Optional<User>

Reply:
ok, done